app/plugin_install.go

```diff
@@ -160,6 +174,11 @@ func (a *App) RemovePlugin(id string) *model.AppError {
 }
 
 func (a *App) removePlugin(id string) *model.AppError {
+	// Disable plugin before removal and notify cluster peers sync.
```

Comment:
Could we elaborate on the `disable` part here? It's obvious that we're disabling, but it's not obvious why.

Also, do we need the comment re: notifying clusters sync? Seems like this comment could be part of the `DisablePlugin` godoc if needed.